core/workflow-activiti/src/main/resources/userWorkflow.bpmn20.xml

```diff
@@ -82,12 +82,47 @@ under the License.
     
     <serviceTask id="delete" name="Delete" activiti:expression="#{delete.execute(execution.processInstanceId)}"/>
     <sequenceFlow id="flow99" sourceRef="delete" targetRef="theEnd"/>
+    <!-- Recertification tasks -->
+    <userTask id="RecertificationRequest" name="Recertification Request" activiti:candidateGroups="managingDirector" activiti:formKey="recertify">
```

Comment:
You cannot refer to the `managingDirector` group in the production workflow, as production content does not contain such group.
Please remove `activiti:candidateGroups` from this element.